desc/objectives/linear_objectives.py

```diff
@@ -3238,3 +3238,218 @@ def compute(self, params, constants=None):
 
         """
         return params["Psi"]
+
+
+class FixNearAxis(_FixedObjective):
+    """Fixes an equilibrium's near-axis behavior.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Equilibrium that will be optimized to satisfy the Objective.
+    nae_eq : Qsc
```

Comment:
it would be good if this was optional, so that you could either fix it to a specific NAE, or just fix it to the current equilibriums near axis behavior